mmv1/products/dialogflow/Generator.yaml

```diff
@@ -0,0 +1,243 @@
+            - name: extraInfo
+              type: KeyValuePairs
+              description: |
+                Optional. Key is the placeholder field name in input, value is the value of the placeholder. E.g. instruction contains "@price", and ingested data has <"price", "10">
+            - name: output
+              type: NestedObject
+              description: |
+                Required. Example output of the model.
+              properties:
+                - name: summarySuggestion
+                  type: NestedObject
+                  description: |
+                    Optional. Suggested summary.
+                  properties:
+                    - type: Array
+                      name: summarySections
+                      description: |
+                        Required. All the parts of generated summary.
+                      item_type:
+                        type: NestedObject
+                        properties:
+                          - type: String
+                            name: section
+                            required: true
+                            description: |
+                              Required. Name of the section.
+                          - type: String
+                            name: summary
+                            required: true
+                            description: |
+                              Required. Summary text for the section.
+            - name: summarizationSectionList
+              type: NestedObject
+              description: |
+                Summarization sections.
+              properties:
+                - type: Array
+                  name: summarizationSections
+                  description: |
+                    Optional. Summarization sections.
+                  item_type:
+                    type: NestedObject
+                    properties:
+                      - type: String
+                        name: key
+                        description: |
+                          Optional. Name of the section, for example, "situation".
+                      - type: String
+                        name: definition
+                        description: |
+                          Optional. Definition of the section, for example, "what the customer needs help with or has question about."
+                      - type: Enum
+                        name: type
+                        description: |
+                          Optional. Type of the summarization section.
+                        enum_values:
+                          - 'SITUATION'
+                          - 'ACTION'
+                          - 'RESOLUTION'
+                          - 'REASON_FOR_CANCELLATION'
+                          - 'CUSTOMER_SATISFACTION'
+                          - 'ENTITIES'
+                          - 'CUSTOMER_DEFINED'
+                          - 'SITUATION_CONCISE'
+                          - 'ACTION_CONCISE'
+      - name: version
```

Comment:
It seems the API return `4.0` for `version` if the field is unset. you'll need to add `default_from_api: true`. See https://googlecloudplatform.github.io/magic-modules/reference/field/#default_from_api and https://googlecloudplatform.github.io/magic-modules/develop/diffs/#default